lib/membership/index.js

```diff
@@ -199,11 +199,66 @@ Membership.prototype.isPingable = function isPingable(member) {
             Member.isStatusPingable(member.status);
 };
 
+/**
+ * Change the status of the local member to alive
+ * @see Membership#setLocalStatus
+ * @see Member.Status.alive
+ */
+Membership.prototype.makeLocalAlive = function makeLocalAlive(){
+    this.setLocalStatus(Member.Status.alive);
+};
+
+/**
+ * Bump the incarnation number of the local member and return an update that
+ * described the current state of the local member.
+ *
+ * @returns {Update} The update to gossip around
+ * @private
+ */
+Membership.prototype._reincarnate = function _reincarnate() {
+    this.localMember.incarnationNumber = Date.now();
```

Comment:
Not sure if you are doing this in the later PRs, but maybe we should make it like this?

```javascript
this.localMember.incarnationNumber = max(this.localMember.incarnationNumber + 1, Date.now());
```